cli/tools/jupyter/install.rs

```diff
@@ -48,13 +49,39 @@ fn install_icon(
   Ok(())
 }
 
-pub fn install() -> Result<(), AnyError> {
-  let temp_dir = TempDir::new().unwrap();
-  let kernel_json_path = temp_dir.path().join("kernel.json");
+pub fn install(directory: Option<PathBuf>) -> Result<(), AnyError> {
+  let outcome = match directory {
+    Some(path) => install_via_directory_flag(&path),
+    None => install_via_jupyter(),
+  };
+
+  if outcome.is_err() {
+    bail!("🆘 Failed to install Deno kernel");
+  }
+
+  println!("✅ Deno kernelspec installed successfully.");
+  Ok(())
+}
+
+fn install_via_directory_flag(output_dir: &PathBuf) -> Result<(), AnyError> {
+  let mut dir = output_dir.clone();
+  if !dir.ends_with("deno") {
+    dir.push("deno");
+  }
```

Comment:
Good for the first pass 👍 for the future we might want to derive the name from the current version - eg. in Deno v1.43 this would add `deno-v1.43` and for Deno v1.44.2 it would add `deno-v1.44`. That would make it very handy managing multiple Deno installations.

Reply:
@bartlomieju I can try it out and confirm it works, if we want it later, may as well start with that behavior.